packages/theme/general.ts

```diff
@@ -1,8 +1,11 @@
 import { StyleSheet } from 'react-native'
 
+import { whiteBackground } from 'theme/include/colors'
+
 export default StyleSheet.create({
   flex: {
-    flex: 1
+    flex: 1,
+    ...whiteBackground
```

Comment:
`flex` is used in many places by things that we don't want to have a white background - it's way too specific for the `general` file and belongs under `account.ts`